nibabel/parrec.py

```diff
@@ -185,8 +185,8 @@
     ('rescale intercept', float),
     ('rescale slope', float),
     ('scale slope', float),
-    ('window center', int,),
-    ('window width', int,),
+    ('window center', float,),
+    ('window width', float,),
```

Comment:
If it was, could you add a comment noting that there is a discrepancy between what is observed in PAR files (integer) and what they actually store (float)? Otherwise the next person might come across this and errantly revert your changes.